code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -90,15 +90,6 @@
 	.=..()
 	handle_charge(H)
 
-/datum/species/ethereal/spec_fully_heal(mob/living/carbon/human/H)
```

Comment:
they still need this behavior, re-implement it with the re-work else admin heal / legion cores don't help ethereals properly

Reply:
Adminheals replace their old stomach with a new one and they get full charge back. I have tested it, it's not needed.

Reply:
Good to hear :)